site/reference/cli/fn/eval/README.md

```diff
@@ -155,6 +159,11 @@ $ kpt fn eval DIR -i gcr.io/example.com/my-fn --fn-config my-fn-config
 $ kpt fn eval DIR -i gcr.io/example.com/my-fn:v1.0.0 -- foo=bar
 ```
 
+```shell
+# execute container my-fn and add it to Kptfile after execution.
+$ kpt fn eval DIR -s -i gcr.io/example.com/my-fn:v1.0.0 -- foo=bar
+```
```

Comment:
So one thing we will have to do is exclude the docs from this PR because as soon as the PR is merged, docs changes become live on the `kpt.dev` and so this reference docs should actually be merged after this change is released to the public.

Reply:
I see. I'll comment them out and add a TODO

Reply:
Here's the other PR https://github.com/GoogleContainerTools/kpt/pull/2848